### ISSUE TITLE
Add exsection 

### ISSUE THREAD

**Issue by emitanaka** [numbats/moodlequiz#10]
exsection: Subject/topic/subtopic

**Comment by mitchelloharawild** [comment]
Please provide more information.
All question (and non-question) quiz types are available in the `type` argument of a question. For example,
```
---
|- type: description
|- name: Background
```

```
---
|- type: essay
|- name: Conclusions
|- defaultgrade: 3
```

**Comment by mitchelloharawild** [comment]
Closed in favour of #24.
